src/lib/global-state/shortcuts-visibility-state.svelte.ts

```diff
@@ -0,0 +1,45 @@
+import { browser } from "$app/environment";
+
+const STORAGE_KEY = "showShortcuts";
+let initialValue = true;
+
+if (browser) {
+    const storedValue = localStorage.getItem(STORAGE_KEY);
+    if (storedValue !== null) {
+        try {
+            const parsedValue = JSON.parse(storedValue);
+            if (typeof parsedValue === "boolean") {
+                initialValue = parsedValue;
+            }
+        } catch (error) {
+            console.error("Error parsing ${STORAGE_KEY} from local storage: ", error);
```

Comment:
Use ` to interpolate strings
```suggestion
            console.error(`Error parsing ${STORAGE_KEY} from local storage:`, error);
```

Reply:
I must have missed this. I will update this.